ee/codegen/src/generators/triggers/integration-trigger.ts

```diff
@@ -88,6 +89,32 @@ export class IntegrationTrigger extends BaseTrigger<IntegrationTriggerType> {
     return body;
   }
 
+  /**
+   * Helper method to create the appropriate AST node for a setup attribute value.
+   * Handles STRING, NUMBER, and null/undefined values.
+   */
+  private createSetupAttributeValue(
```

Comment:
DevinAI don't we already have a generator for VellumValue that we can use instead?